great_expectations/analytics/actions.py

```diff
@@ -3,3 +3,9 @@
 from great_expectations.analytics.base_event import Action
 
 DATA_CONTEXT_INITIALIZED = Action("data_context.initialized")
+EXPECTATION_SUITE_CREATED = Action("expectation_suite.created")
+EXPECTATION_SUITE_DELETED = Action("expectation_suite.deleted")
+EXPECTATION_SUITE_UPDATED = Action("expectation_suite.updated")
+EXPECTATION_SUITE_EXPECTATION_CREATED = Action("expectation_suite.expectation_created")
+EXPECTATION_SUITE_EXPECTATION_DELETED = Action("expectation_suite.expectation_deleted")
+EXPECTATION_SUITE_EXPECTATION_UPDATED = Action("expectation_suite.expectation_updated")
```

Comment:
Not sure if all these will be used but simply going off of initial discussions and the analytics doc we have in Confluence

Reply:
Should the latter 3 be just `Action("expectation.<CRUD>")`?

Reply:
Hmm based on the Confluence doc I think we're not doing `expectation.<CRUD>` but rather `expectation_suite.expectation_<CRUD>`